src/routing/pages/probe/FirefoxLegacyDetails.svelte

```diff
@@ -0,0 +1,236 @@
+      'https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/collection/index.html',
+};
+</script>
+
+<style>
+  .drawer-section {
+    padding: var(--space-2x) 0;
+  }
+  .drawer-section--end {
+    align-self: end;
+    min-height: calc(var(--increment) * 2);
+    position: grid;
+    grid-auto-flow: row;
+    grid-row-gap: var(--space-base);
+  }
+
+  h2 {
+    padding-bottom: var(--space-base);
+  }
+
+  .probe-description {
+    color: var(--subhead-gray-02);
+    word-break: break-word;
+  }
+
+  .bug-list {
+    display: flex;
+    flex-wrap: wrap;
+    line-height: 1.8;
+  }
+
+  .bug-list a {
+    display: block;
+    margin-right: var(--space-base);
+  }
+  .probe-details-overview-left {
+    margin: 0;
+  }
+
+  .probe-details-overview-left > * {
+    line-height: 1.2;
+  }
+
+  .probe-details-overview-left dt {
+    text-transform: capitalize;
+    color: var(--subhead-gray-01);
+    font-size: var(--text-04);
+    font-weight: 500;
+  }
+
+  .probe-details-overview-left dd {
+    margin: 0;
+    font-size: var(--text-015);
+    color: var(--subhead-gray-01);
+  }
+
+  .probe-details-overview-left--padded {
+    padding: 0 calc(var(--space-base) - 1px);
+  }
+
+  .probe-details-overview-left--subtle {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 0;
+    align-items: baseline;
+  }
+
+  .probe-details-overview-left--subtle dt {
+    text-transform: none;
+    font-size: var(--text-02);
+  }
+
+  .probe-details-overview-left--subtle dd {
+    font-size: var(--text-01);
+  }
+
+  .probe-details-overview {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .probe-details-overview-right {
+    padding: var(--space-2x) 0;
+  }
+
+  .more-info-link {
+    display: grid;
+    align-items: center;
+    grid-auto-flow: column;
+    grid-column-gap: var(--space-1h);
+    justify-content: end;
+    font-weight: 300;
+    padding-right: var(--space-1h);
+    margin: var(--space-2x) 0;
+    font-size: var(--text-01);
+  }
+
+  .probe-type-link {
+    color: var(--subhead-gray-01);
+  }
+
+  /* FIXME: once @graph-paper/button supports href, use the Button component. */
+  .docs-button {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: transparent;
+    display: grid;
+    width: 100%;
+    color: var(--digital-blue-500);
+    border: none;
+    cursor: pointer;
+    display: grid;
+    grid-auto-flow: column;
+    align-items: center;
+    text-decoration: none;
+    justify-content: center;
+    grid-column-gap: var(--space-base);
+    text-transform: uppercase;
+    font-size: var(--text-015);
+    font-weight: 500;
+    padding: var(--space-base);
+    transition: background-color 100ms;
+    border-radius: var(--space-1h);
+  }
+
+  .docs-button:hover {
+    background-color: var(--cool-gray-150);
+  }
+</style>
+
+<!-- probe-details-content -->
+<div class="probe-details-content">
+  <div class="probe-details-overview">
+    {#if $probe.type}
```

Comment:
> Uncaught (in promise) TypeError: can't access property "type", ctx[0] is undefined

## Steps to reproduce

1. Visit http://localhost:5000/firefox/probe/gc_ms/explore
2. Click on the GLAM logo to go home